src/Axiom/Set/Properties.agda

```diff
@@ -100,6 +107,28 @@ cong-⊆⇒cong₂ h X≡ᵉX' Y≡ᵉY' = h (proj₁ X≡ᵉX') (proj₁ Y≡
   { isPreorder = ⊆-isPreorder
   ; antisym    = _,_ }
 
+∈-× : ∀{a : A} {b : B} → (a , b) ∈ X → (a ∈ map proj₁ X × b ∈ map proj₂ X)
```

Comment:
Remove `∀` in all of these or remove the type annotations for the things under `∀`

Reply:
Also, leave a space after `∀`

Reply:
There are also a few more places to apply this to